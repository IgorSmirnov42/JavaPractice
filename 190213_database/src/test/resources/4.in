1
a
2
4
a
2
4
a
3
7
0
