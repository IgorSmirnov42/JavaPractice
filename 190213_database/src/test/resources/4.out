Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write command name (8 for help): Write owner name: Write command name (8 for help): Unknown command name
Write command name (8 for help): Write phone number: Write command name (8 for help): 